condor/condor_meter

```diff
@@ -774,7 +774,11 @@ def htcondor_configured():
     """ 
     try:
         path = GratiaCore.Config.getConfigAttribute("CondorLocation")
-        condor_config_path = os.path.join(path, "bin", "condor_config_val")
+        if 'htcondor-ce' in g_probe_config and g_probe_config is not None:
+          condor_binary = 'condor_ce_config_val'
```

Comment:
`condor_config_binary` would be more descriptive

Reply:
renamed